model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -125,8 +125,10 @@ def restore_correct_ports(graph: Graph):
         if 'out' in d:
             node = Node(graph, u)
             num_of_in_nodes = len(node.in_nodes())
-            # we need to check operation type, if it is const op, we don't renumber out ports
-            decremented_number = d['out'] - num_of_in_nodes if node.type != 'Const' else d['out']
+            decremented_number = d['out'] - num_of_in_nodes
+            # we need to check operation type, if it is const op with out_port == 0, we don't renumber out ports
+            if node.type == 'Const' and d['out'] == 0:
+                decremented_number = d['out']
```

Comment:
The changes of this file should not be a part of this PR.

Reply:
Ok, removed.